alerts.go

```diff
@@ -43,26 +44,66 @@ type Alert struct {
 	ClosedAt  int64   `json:"closedAt,omitempty"`
 }
 
-// FindAlerts find alerts
-func (c *Client) FindAlerts() ([]*Alert, error) {
-	req, err := http.NewRequest("GET", c.urlFor("/api/v0/alerts").String(), nil)
+// NextID is next id of alert
+type NextID string
+
+// Data includes alert and next id
+type Data struct {
```

Comment:
We need more detailed naming to express data *of Alerts API*, since mackerel-client-go has only one namespace.